test/python/fe_api/causal_conv1d_bulk/test_causal_conv1d_bulk_sm100.py

```
@@ -0,0 +1,563 @@
+    # rather than trusting the descriptors used for compilation.
+    with pytest.raises(ValueError, match="Output T must match X"):
+        api.execute(
+            x,
+            weight,
+            torch.empty(batch, tokens + 1, channels, device="cuda", dtype=x.dtype),
+            initial_state_tensor=initial_state,
+            final_state_tensor=final_state,
+        )
+    with pytest.raises(TypeError, match="Output dtype mismatch"):
+        api.execute(x, weight, output.float(), initial_state_tensor=initial_state, final_state_tensor=final_state)
+    with pytest.raises(ValueError, match="Output must be.*contiguous"):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)